app2/www/js/chatinfo/chatinfo.js

```diff
@@ -1,6 +1,45 @@
 'use strict';
 angular.module('crewapp.chatinfo', [])
-.controller('ChatInfoController', function($scope){
+.controller('ChatInfoController', function($scope, plansFactory){
   $scope.test = 'hello';
+    $scope.bothEntered = true;
 
-});
+    $scope.plans = plansFactory;
+
+    $scope.makePlans = function(place) {
+      var photo = function() {
+        if (!!place.photos){
+          return place.photos[0].getUgrl({'maxWidth': 150, 'maxHeight': 150});
+        } else {
+          return 'http://www.mtwomeybutchers.ie/wp-content/uploads/placeholder.gif';
+        }
+      }
+      if (!$scope.one) {
+        $scope.one = {};
+        $scope.one.photo = photo();
+        $scope.one.name = place.name;
+        $scope.one.address = place.formatted_address
+        $scope.one.numVotes = 0;
+        console.log(place);
+      }
+      else if (!$scope.secondPlace) {
+        $scope.two = {};
+        $scope.two.photo = photo();
+        $scope.two.name = place.name;
+        $scope.two.address = place.formatted_address
+        $scope.two.numVotes = 0;
+        $scope.bothEntered = false;
+      }
+    }
+
+    $scope.submitPlans = function() {
+      $scope.plans.one = $scope.one;
+      console.log($scope.one);
+      $scope.plans.two = $scope.two;
+      console.log($scope.two);
+    }
+  })
+  .factory('plansFactory', function() {
```

Comment:
Right now it's saving the object created in the controller over to the Factory which will in turn be used by the ChatRoom.

It's being done this way because if a user wants to suggest multiple events, the data doesn't persist through to the controller past view changes so I can suggest new events.